include/fmt/ranges.h

```diff
@@ -11,9 +11,13 @@
 #ifndef FMT_IMPORT_STD
 #  include <initializer_list>
 #  include <iterator>
+#  include <string>
 #  include <tuple>
 #  include <type_traits>
 #  include <utility>
+#  ifdef __cpp_lib_ranges
```

Comment:
@vitaut 
It might be better to check like this:
https://github.com/fmtlib/fmt/blob/c4ea903250e20c3ffa9b70695c33207dc9b9a0e2/include/fmt/std.h#L46-L48

```c++
 #  if FMT_HAS_INCLUDE(<ranges>) && FMT_CPLUSPLUS >= 202002L 
 #    include <ranges> 
 #  endif
```

?

Reply:
To avoid implicit dependencies.

``__cpp_lib_ranges`` is defined in headers:

- ``<algorithm>``
- ``<functional>``
- ``<iterator>``
- ``<memory>``
- ``<ranges>``

https://en.cppreference.com/w/cpp/feature_test#cpp_lib_ranges

Reply:
I think we can drop the dependency here but it can be done separately.